tests/specifications/optional-request-body.yaml

```diff
@@ -0,0 +1,223 @@
+openapi: '3.0.0'
```

Comment:
We will continue to maintain the tests, so we would like to keep them to a minimum as much as possible. Is it possible to slim down to a minimized case that only checks the request body?

Reply:
Absolutely, I'll fix!